src/bindings/glib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	target_compile_options (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_CFLAGS_OTHER})
 
 	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC elektra-core elektra-kdb)
-	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LDFLAGS})
+	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LIBRARIES})
```

Comment:
```
<XPREFIX>_LIBRARIES      ... only the libraries (w/o the '-l')
<XPREFIX>_LDFLAGS        ... all required linker flags
```

is there really a difference other than the -l? I mean they both just use pkg-config information.
Are you sure this was not just a workaround for OS X using brew cellar components. Look at the current output it shows both variables on osx.

```
CMake Warning at src/bindings/glib/CMakeLists.txt:38 (message):
  GOBJECT LIBRARIES: gobject-2.0glib-2.0intl
CMake Warning at src/bindings/glib/CMakeLists.txt:39 (message):
  GOBJECT LDFLAGS:
  -L/usr/local/Cellar/glib/2.46.2/lib-L/usr/local/opt/gettext/lib-lgobject-2.0-lglib-2.0-lintl
```